templates/taxi/car_delete.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete {{ car.model }}</h1>
+  <p>Are you sure that you want to delete {{ car.model }}</p>
+  <p><i>
+    If you delete {{ car.model }}, all other elements which content this model will be also deleted
+  </i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Delete">

```

Comment:
According to the checklist, you should add a 'Cancel' button apart from the 'Delete' one. The 'Cancel' button should lead to the previous page the user was on.